plugins/lightspeed-backend/src/service/types.ts

```diff
@@ -2,10 +2,10 @@ import {
   // DiscoveryService,
```

Comment:
Is there a reason why the types are commented out?

Reply:
I guess @yangcao77 added these for future reference? https://github.com/janus-idp/backstage-plugins/blob/main/plugins/lightspeed-backend/src/plugin.ts#L22-L24
I can remove them if @yangcao77 is ok with it.

Reply:
I will  those for user authentication, but feel free to remove it if you would like to.